client/template.go

```diff
@@ -78,7 +78,7 @@ type TemplateCreatePayload struct {
 	GitlabProjectId      int              `json:"gitlabProjectId,omitempty"`
 	Revision             string           `json:"revision"`
 	OrganizationId       string           `json:"organizationId"`
-	TerraformVersion     string           `json:"terraformVersion"`
+	TerraformVersion     string           `json:"terraformVersion,omitempty"`
```

Comment:
Hi @TomerHeber , I'm Sagy from RND, nice to github-meet you  😃 
on purpose you don't have the `tfschema:",omitempty"` syntax here also?

Reply:
Hi @sagydr - nice to meet you (:

tfschema omitempty is only needed when reading from a struct and writing to the terraform schema.
For this struct we read from the terraform schema and write to the struct.

When transforming to a json, the json omitempty will be used and the field will not be sent.
(if string == "" then the field isn't passed in the json).

Happy to explain more if this is still unclear. (Didn't want the explanation to be too long).